src/main/scala/com/fulcrumgenomics/bam/FilterBam.scala

```diff
@@ -62,7 +68,11 @@ class FilterBam
   @arg(flag='D', doc="If true remove all reads that are marked as duplicates.")   val removeDuplicates: Boolean = true,
   @arg(flag='U', doc="Remove all unmapped reads.")                                val removeUnmappedReads: Boolean = true,
   @arg(flag='M', doc="Remove all mapped reads with MAPQ lower than this number.") val minMapQ: Int = 1,
-  @arg(flag='S', doc="Remove all reads marked as secondary alignments.")          val removeSecondaryAlignments: Boolean = true
+  @arg(flag='P', doc="Remove all reads that are not part of a mapped pair.")      val removeUnpairedReads: Boolean = false,
+  @arg(flag='S', doc="Remove all reads marked as secondary alignments.")          val removeSecondaryAlignments: Boolean = true,
+  @arg(          doc="Remove all reads with insert size < this value.")           val minInsertSize: Option[Int] = None,
+  @arg(          doc="Remove all reads with insert size > this value.")           val maxInsertSize: Option[Int] = None,
+  @arg(flag='m', doc="Remove reads with fewer than this many mapped bases.")      val minMappedBases: Option[Int] = None
 ) extends FgBioTool with LazyLogging {
 
```

Comment:
Can you add a warning when `--remove-unmapped-reads=false` but one of the other options overrides it?